### ISSUE TITLE
The documentation on spouts doesn't mention composer autoload

### ISSUE THREAD

**Issue by tuur29** [SSilence/selfoss#958]
I was getting the following generic error: `Class spouts\privaterss\privaterss does not exist ` when I tried adding my own spout. After an hour I found that you have to regenerate the composer autoload files. This isn't mentioned on the website yet.

**Comment by jtojnar** [comment]
Sorry about that, I wanted to use PSR-4 to eliminate the need for autoload dumping but for some reason concluded that file structure would need to be changed and then forgot about it completely. Since the problem does not seem to exist, I migrated selfoss to the standard.

It will be fixed in #959, then initial autoloader generated with `composer install` or provided with package should suffice.

As for documentation, #957 should take care of it.

**Comment by jantman** [comment]
FWIW, I think this still needs further documentation beyond #957. I'm lucky I found this, because I've been banging my head against the wall for a few hours.

I'm not installing from git, I'm actually using a [Docker image](https://hub.docker.com/r/hardware/selfoss/). I followed the documentation under "Extend" on https://selfoss.aditu.de/ which implied that I could just copy ``/spouts/rss/feed.php`` to ``/spouts/jantman/feed.php``, make the relevant changes, and restart the container. I think the docs on that website need to be updated.
